index.html

```diff
@@ -0,0 +1,158 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+  <!-- Google fonts -->
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700;900&display=swap" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600;700;800;900&display=swap" rel="stylesheet">
+
+  <!-- Css links -->
+  <link rel="stylesheet" href="./style.css">
+
+  <!-- JavaScipt links -->
+
+  <title>Document</title>
+</head>
+
+<body>
+  <header class="page-header column-center">
+    <button type="button" name="button" class="menu">
+      <img src="./images/buttons/hamburger.svg" alt="menu">
```

Comment:
- Your hamburger button isn't displaying the menu. Kindly work on it.
- Fix the button position.
<img width="1440" alt="Screen Shot 2021-09-13 at 19 01 40" src="https://user-images.githubusercontent.com/22258274/133172419-1c9f2dd6-fd9f-4a94-95cf-ac5fc45e94a1.png">